source/_docs/pantheon-yml.md

```diff
@@ -111,6 +72,25 @@ Add `drush_version` to the top level of the `pantheon.yml` file to configure the
 drush_version: 8
 ```
 For more information, see [Managing Drush Versions on Pantheon](/docs/drush-versions).
+
+
+### Custom Upstreams and pantheon.upstream.yml
+Add a `pantheon.upstream.yml` file to your organization's custom upstream to set default settings for all downstream sites. Downstream sites can override the defaults at the site-level with pantheon.yml. Quicksilver workflows are not supported in pantheon.upstream.yml.
+
+Create the `pantheon.upstream.yml` in the root of the custom upstream's codebase. Help avoid merge conflicts by adding pantheon.upstream.yml to the upstream's .gitignore. Example: [Adding pantheon.upstream.yml to Drops-7](https://github.com/pantheon-systems/WordPress/pull/123/files)
```

Comment:
I can clean this bit up in a copy review, but I think Drops-7 should be WordPress here. Also, looks like we're already ignoring this by default in our upstream. So if folks follow the recommended path when creating custom upstreams, it should already ignore this file. Might not need to include instructions to ignore it now